app/controllers/AssetWebApi.scala

```diff
@@ -9,6 +9,7 @@ import util.concurrent.BackgroundProcessor
 
 import play.api.mvc._
 import play.api.libs.json._
+import play.api.libs.concurrent.Execution.Implicits._
```

Comment:
AssetCancelProcessor needs an ExecutionContext since it is using Scala futures/Awaits to handle the Softlayer.cancelServer call.